CONTRIBUTING.md

```diff
@@ -115,15 +189,15 @@ and hence must be used with caution.
 ## Use Cases
 
 - Configure scrape targets through Juju relations.
-- Enable alerting through a relation with Alertmanager.
+- Configure alerting rules through relations with scrape target charms.
+- Enable alert forwarding through a relation with Alertmanager.
 - Support metrics visualisation through Grafana.
 
 ## Road Map
 
 Roughly by order of priority
```

Comment:
You know, I'd remove the entire section. The roadmap is something we will eventually discuss on Discourse, has no place IMO in a Git history.

Reply:
OK. Will do.

Reply:
Done.